### DISCUSSION TITLE
My work on Captionfy.com

### DISCUSSION THREAD

**Discussion by n-endashi** [machinewrapped/llm-subtrans#95] [Show and tell]
I recently discovered this website so I combined to sub Korone's latest stream using whisper and gpt-4 turbo with gpt-subtrans. Check it out!
https://www.captionfy.com/video/youtube/YObmKx80cRk?caption=657da6f395dd8fa734d72610

**Comment by machinewrapped** [comment]
Haha, I was a bit worried by the caption for the opening music, but once she starts speaking it looks good - still no idea what I'm watching, but at least the subtitles make sense!
